src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -91,6 +91,12 @@ export const monoPlatform: Platform = {
     return mono_string(jsString);
   },
 
+  getByteArray: function getByteArray(array: System_Array<any>): Uint8Array {
```

Comment:
For consistency with the other value-converting method (`toJavaScriptString`), could this be renamed to `toUint8Array`?